node_modules/nodeos-barebones/node_modules/nodeos-cross-toolchain/scripts/build

```diff
@@ -112,6 +112,8 @@ if [[ ! -d $OBJ_DIR ]]; then
         --with-sysroot=$TOOLS/$TARGET         \
         --disable-nls                         \
         --disable-shared                      \
+        --disable-target-libsanitizer         \
+        --disable-libsanitizer                \
```

Comment:
that link is the email which refers to this commit: http://git.buildroot.net/buildroot/commit/?id=5f4d658d888b539de9a6247ae5b1a0999de5d4ec